backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java

```diff
@@ -659,14 +547,136 @@ public void setCursor (Cursor cursor) {
 	public void setSystemCursor (SystemCursor systemCursor) {
 	}
 
+	public static class IOSUIViewController extends GLKViewController {

```

Comment:
Please extract this to an own class. It serves not only for graphics, but also for input, and needs to be overriden in game projects often. The only reason to keep it an inner class is to keep the git history, but we erase it with this move so no reason to keep it inner any more.

Reply:
Done. The reason was not to break backwards compatibility on this PR but, considering the change makes sense and that  `IOSUIViewController` deserves a refactor, I think it's a good opportunity to move it to its own class now and apply the refactor on the new class later so that it's easier to review.